helm/values-dev.yaml

```diff
@@ -85,10 +85,12 @@ microservice-chart:
     ELASTIC_APM_LOG_SENDING: "false"
     ECS_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
     ECS_SERVICE_ENVIRONMENT: "dev"
+    PERSONAL_DATA_VAULT_API_BASE_PATH: "https://api.uat.tokenizer.pdv.pagopa.it/tokenizer/v1"
```

Comment:
Is it a typo or we use uat pdv in dev?

Reply:
use the same pdv test environment (api.uat.tokenizer.pdv.pagopa.it) for dev and uat (so api.dev.platform.pagopa.it and api.uat.platform.pagopa.it)